bin/optimise-gene2experiment.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
```

Comment:
This script looks basically identical to the other, is there a reason for having two scripts?

Reply:
I think the same happens in the other creation and loading scripts...